spec/robots/accession/embargo_release_spec.rb

```diff
@@ -0,0 +1,133 @@
+      </embargoMetadata>
+      EOXML
+    }
+    let(:item) {
+      i = Dor::Item.new
+      rds = Dor::RightsMetadataDS.new
+      rds.content = Nokogiri::XML(rights_xml) {|config| config.default_xml.noblanks}.to_s
+      i.datastreams['rightsMetadata'] = rds
+      eds = Dor::EmbargoMetadataDS.new
+      eds.content = Nokogiri::XML(embargo_xml) {|config| config.default_xml.noblanks}.to_s
+      i.datastreams['embargoMetadata'] = eds
+      i
+    }
+    it 'rights metadata has no embargo after Dor::Item.release_embargo' do
+      expect(item.rightsMetadata.ng_xml.at_xpath('//embargoReleaseDate')).not_to be_nil
+      expect(item.rightsMetadata.content).to match('embargoReleaseDate')
+      item.release_embargo('ignored')
+      expect(item.rightsMetadata.ng_xml.at_xpath('//embargoReleaseDate')).to be_nil
+      expect(item.rightsMetadata.content).not_to match('embargoReleaseDate')
+    end
+
+    it 'embargo metadata changes to status released after Dor::Item.release_embargo' do
+      expect(item.embargoMetadata.ng_xml.at_xpath('//status').text).to eql 'embargoed'
+      item.release_embargo('ignored')
+      expect(item.embargoMetadata.ng_xml.at_xpath('//status').text).to eql 'released'
+    end
+  end
+
+  context 'release_20_pct_vis_embargo' do
+    let(:embargo_twenty_pct_xml) { <<-EOXML
+      <embargoMetadata>
+        <status>embargoed</status>
+        <releaseDate>#{embargo_release_date.iso8601}</releaseDate>
+        <twentyPctVisibilityStatus>anything</twentyPctVisibilityStatus>
+        <twentyPctVisibilityReleaseDate>#{embargo_release_date.iso8601}</twentyPctVisibilityReleaseDate>
+        #{release_access}
+      </embargoMetadata>
+      EOXML
+    }
+    let(:item) {
+      i = Dor::Item.new
+      rds = Dor::RightsMetadataDS.new
+      rds.content = Nokogiri::XML(rights_xml) {|config| config.default_xml.noblanks}.to_s
+      i.datastreams['rightsMetadata'] = rds
+      eds = Dor::EmbargoMetadataDS.new
+      eds.content = Nokogiri::XML(embargo_twenty_pct_xml) {|config| config.default_xml.noblanks}.to_s
+      i.datastreams['embargoMetadata'] = eds
+      i
+    }
+
+    it 'rights metadata has no embargo after Dor::Item.release_20_pct_vis_embargo' do
+      expect(item.rightsMetadata.ng_xml.at_xpath('//embargoReleaseDate')).not_to be_nil
+      expect(item.rightsMetadata.content).to match('embargoReleaseDate')
+      item.release_20_pct_vis_embargo('ignored')
+      expect(item.rightsMetadata.ng_xml.at_xpath('//embargoReleaseDate')).to be_nil
+      expect(item.rightsMetadata.content).not_to match('embargoReleaseDate')
+    end
+
+    it 'embargo metadata changes to twenty_pct_status released after Dor::Item.release_20_pct_vis_embargo' do
+      expect(item.embargoMetadata.twenty_pct_status).to eql 'anything'
+      item.release_20_pct_vis_embargo('ignored')
+      expect(item.embargoMetadata.twenty_pct_status).to eql 'released'
+    end
+  end
+
+  it 'running embargo_release.rb attempts to run EmbargoRelease.release' do
+
```

Comment:
wontfix, will remove (couldn't come up with a way to test that didn't require more refactoring, didn't seem worth expending tons of effort on since it'll be pretty obvious from trying to run the script whether it's doing something or nothing)